cico_functions.sh

```diff
@@ -44,6 +68,15 @@ function install_deps() {
     git
 
   service docker start
+  
+  #set buildx env
+  export DOCKER_BUILD_KIT=1
+  export DOCKER_CLI_EXPERIMENTAL=enabled
+
+  #Enable qemu and binfmt support
+  docker run --rm --privileged docker/binfmt:66f9012c56a8316f9244ffd7622d7c21c1f6f28d
```

Comment:
is this tag expected to be used?

Reply:
Yes @ibuziuk The tag is expected to be used. Available tags for the image docker/binfmt can be found [here].(https://hub.docker.com/r/docker/binfmt/tags)  
Also, updated the tag to the latest one.